modules/ossm-add-project-using-label-selectors-cli.adoc

```diff
@@ -0,0 +1,52 @@
+// Module included in the following assemblies:
+//
+// * service_mesh/v2x/installing-ossm.adoc
+
+:_content-type: PROCEDURE
+[id="ossm-adding-project-using-label-selectors-cli_{context}"]
+= Adding a project to the {SMProductShortName} using label selectors with the CLI 
+
+You can use labels selectors to add a project to the {SMProductShortName} with the CLI.
+
+.Prerequisites
+* An installed, verified {SMProductName} Operator.
+* An existing `ServiceMeshMemberRoll` resource.
+* To perform this task, a user must have mesh admin privileges.
+
+.Procedure
+
+. Log in to the {product-title} CLI.
+
+. Edit the `ServiceMeshMemberRoll` resource.
++
+[source,terminal]
+----
+$ oc edit smmr -n <controlplane_project>
+----
++
+The example abouve uses `<controlplane_project>` as an example. You can deploy the {SMProductShortName} control plane to any project as long as it is separate from the project that contains your services.
+
+. Modify the YAML file to include namespace label selectors in the `spec.memberSelectors` field of the `ServiceMeshMemberRoll` resource.
++
+[NOTE]
+====
+Instead of using the `matchLabels` field, you can also use the `matchExpressions` field in the selector.
+====
++
+[source,yaml]
+----
+apiVersion: maistra.io/v1
+kind: ServiceMeshMemberRoll
+metadata:
+  name: default
+  namespace: istio-system
+spec:
+  memberSelectors: <1>
+  - matchLabels: <2>                 
```

Comment:
```suggestion
  - matchLabels:                
```
- I don't think it's necessary to have multiple callout references in this sample

Reply:
When the developer and I worked through this, we thought it was best to include these callouts. I'd like to keep them.